connect-inject/container_init.go

```diff
@@ -21,6 +21,7 @@ type initContainerCommandData struct {
 	// WriteServiceDefaults controls whether a service-defaults config is
 	// written for this service.
 	WriteServiceDefaults bool
+	ConsulNamespace      string
```

Comment:
Can you add a doc for this and also how if it's empty that means we're not using namespaces

Reply:
Added.